docs/content/stepbystep/Git-setup.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Git setup'
+metaTitle: 'Installing and using git | Discordeno'
+metaDescription: 'Setting up git for your pc'
+---
+
+# Installing
+
+## Installing the CLI
+
+We will now be installing the CLI. This is the terminal interface for git.
+
+First, go to the [Git download page](https://git-scm.com/downloads) and install the version for your OS. I'm using windows so I'll use <https://git-scm.com/download/win>
```

Comment:
> I'm using windows so I'll use <https://git-scm.com/download/win>

Is this really needed?

Reply:
oh yeah, those are extras i forgot to remove

Reply:
how are the new changes